# Description: Compile in our shim to allow Oxide to use non-exported
#  GPU related APIs from content
# Author: Chris Coulson <chris.coulson@canonical.com>

diff --git a/content/content_common.gypi b/content/content_common.gypi
--- a/content/content_common.gypi
+++ b/content/content_common.gypi
@@ -502,16 +502,18 @@
       'common/view_messages.h',
       'common/webplugin_geometry.cc',
       'common/webplugin_geometry.h',
       'common/websocket.cc',
       'common/websocket.h',
       'common/websocket_messages.h',
       'common/worker_messages.h',
       'common/zygote_commands_linux.h',
+      '<(DEPTH)/../../../shared/port/content/common/gpu_thread_shim_oxide.cc',
+      '<(DEPTH)/../../../shared/port/content/common/gpu_thread_shim_oxide.h',
     ],
   },
   'sources': [
     '<@(public_common_sources)',
     '<@(private_common_sources)',
   ],
   'target_conditions': [
     ['OS=="android" and <(use_seccomp_bpf)==1', {
